### DISCUSSION TITLE
Catppuccin

### DISCUSSION THREAD

**Discussion by er2de2** [excalith/excalith-start-page#30] [Showcase]
My catppuccin startpage config

![Screenshot 2023-04-13 at 22-28-37 roman Start Page](https://user-images.githubusercontent.com/62346235/231877125-e327c28b-3713-4a45-b3d4-35defb94ad9a.png)
![Screenshot 2023-04-13 at 22-28-18 roman Start Page](https://user-images.githubusercontent.com/62346235/231877179-4a10640b-53b6-481f-beba-f0ad35d646a4.png)
![Screenshot 2023-04-14 at 07-56-55 roman Start Page](https://user-images.githubusercontent.com/62346235/231954424-69c73398-2c4b-4aeb-ae48-c31ba3b297d1.png)

**Comment by excalith** [comment]
This is so cute! Loved the fetch image btw
